src/main/java/frc/robot/RobotContainer.java

```diff
@@ -24,15 +26,34 @@
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
   private final DriveSystem driveSystem;
-
-  private final XboxController driver = new XboxController(OperatorConstants.kDriverControllerPort);
+  private final GripperSystem gripperSystem;
+  private final Limelight limelight;
+  private final XboxController driverController;
+  private final XboxController operatorController;
```

Comment:
are we using driver xbox this year? i have a pr from sushil that says driver controller is a joystick

Reply:
dont mark as resolved w/out addressing comments